modules/bbp-core/src/main/resources/schemas/bbp/core/entity/v0.1.0.json

```diff
@@ -6,6 +6,9 @@
     }
   ],
   "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/nexus/core/distribution/v0.1.0"
```

Comment:
Should we make mediaType optional in the case of accessURL?

Reply:
Media is optional for both accessURL and downloadURL.